docs/explanation/charmed-rules.rst

```diff
@@ -0,0 +1,39 @@
+Charmed alert rules
+*******************
+
+What are alert rules?
+=====================
+
+Prometheus and Loki would post alerts to Alertmanager when alert expressions are
+evaluated as "true". Alert expressions are stored in yaml files called "alert rules".
```

Comment:
This is a good start, but maybe I'd expand a bit on what they are without mentioning any technology. 

Then, when mentioning Prometheus/Loki/Alertmanager, I would make it more clear that Alertmanager is only responsible for the “notification” part, and the evaluation happens in Prometheus. 

It could even be as simple as saying something like this:
> Prometheus and Loki evaluate the alerts and, if any is triggered, they send the details to Alertmanager, which then takes care of notifying the user.

Reply:
Agreed. Re alertmanager, it seems slightly off-topic, especially since "alerts triggered" can be consumed from prometheus and grafana as well.